OrbitClientGgp/ClientGgp.h

```diff
@@ -40,12 +41,16 @@ class ClientGgp final : public CaptureListener {
   void OnCallstackEvent(orbit_client_protos::CallstackEvent callstack_event) override;
   void OnThreadName(int32_t thread_id, std::string thread_name) override;
   void OnAddressInfo(orbit_client_protos::LinuxAddressInfo address_info) override;
+  void OnUniqueTracepointInfo(uint64_t key,
+                              orbit_grpc_protos::TracepointInfo tracepoint_info) override;
+  void OnTracepointEvent(orbit_client_protos::TracepointEventInfo tracepoint_event_info) override;
 
  private:
   ClientGgpOptions options_;
   std::shared_ptr<grpc::Channel> grpc_channel_;
   std::shared_ptr<Process> target_process_;
   std::shared_ptr<StringManager> string_manager_;
+  std::shared_ptr<TracepointInfoManager> tracepoint_info_manager_;
```

Comment:
Is there a good reason to have TracepointInfoManager stored in a shared_ptr and not an unique_ptr?

This confused me at first (I thought OrbitApp and ClientGgp are sharing the same instance - which is not the case).

I see that others are using shared_ptr as well. So please keep as is if the shared ownership is intended and needed further down the road.